src/components/v2v3/V2V3Project/V2V3ProjectHeaderActions/V2V3ProjectHeaderActions.tsx

```diff
@@ -21,11 +22,14 @@ export function V2V3ProjectHeaderActions() {
 
   const { projectId } = useContext(ProjectMetadataContext)
 
+  if (projectId === undefined) return null
+
   return (
     <>
       <div className="flex items-center">
         <div className="flex items-center gap-x-4">
-          <div>
+          <SubscribeButton projectId={projectId} tooltipPlacement={'bottom'} />
+          <>
```

Comment:
is this needed?

Reply:
unsure - it was an empty div so I decided to just turn into a fragment, proabbyl dont need though